modules/ROOT/pages/deploy-on-premises.adoc

```diff
@@ -213,6 +215,33 @@ include::example$mmp-concept-config.xml[]
 mvn clean package deploy -DmuleDeploy
 ----
 
+== Deploy Applications in Parallel
+
+You can deploy applications in parallel to on-premises Mule instances. Deploying applications in parallel reduces the startup time when you deploy a large number of apps. To prevent deployment failure, ensure that applications are not dependent on each other because a particular start order cannot be guaranteed.
+
+To enable parallel deployment:
+
+. Package or export your Mule applications to the `/apps` directory of your Mule runtime engine instance.
+. Start Mule using the `-M-Dmule.deployment.parallel` option:
++
+```
+mule -M-Dmule.deployment.parallel
+```
+
+After performing these steps, all applications in the `/apps` directory start in parallel.
+
+== Update Applications at Runtime
+
+Updating a Mule application at runtime can be a complex change involving class modifications, endpoint modifications (for example, changing ports, etc), as well as reconfiguring flows. As a result, any application update does a graceful app shutdown under the hood and reconfigures itself. In practice, this is pretty transparent to the user and happens within seconds.
```

Comment:
```suggestion
Updating a Mule application at runtime can be a complex change involving class modifications, endpoint modifications (for example, changing ports), and reconfiguring flows. To keep this process from disrupting users, an application update first performs a graceful app shutdown and then reconfigures itself transparently and within seconds.
```
Or similar, if this alters intended meaning. The troublesome phrase was "as a result," because I think that we want to refer to the transparency as the result and the shutdown/reconfigure as the way to address the complexity?